mysqli/mysqli.php

```diff
@@ -2070,16 +2081,16 @@ function mysqli_query ($link, $query, $resultmode = MYSQLI_STORE_RESULT) {}
  * @link https://php.net/manual/en/mysqli.real-connect.php
  * @see mysqli_connect()
  * @param mysqli $link A link identifier returned by mysqli_connect() or mysqli_init()
- * @param string $host
- * @param string $user
- * @param string $password
- * @param string $database
- * @param string $port
- * @param string $socket
- * @param int $flags
+ * @param string $host [optional]
+ * @param string $user [optional]
+ * @param string $password [optional]
+ * @param string $database [optional]
+ * @param int $port [optional]
+ * @param string $socket [optional]
+ * @param int $flags [optional]
  * @return bool
  */
-function mysqli_real_connect ($link, $host = '', $user = '', $password = '', $database = '', $port = '', $socket = '', $flags = null) {}
+function mysqli_real_connect ($link, $host = '', $user = '', $password = '', $database = '', $port = 0, $socket = '', $flags = null) {}
```

Comment:
According to https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.stub.php#L651 default values for options parameters are nulls. So I think it makes sense to update them accordingly.